src/components/Button.jsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+const Button = styled.button`
+  background: ${(props) => (props.primary ? 'palevioletred' : 'white')};
+  color: ${(props) => (props.primary ? 'white' : 'palevioletred')};
```

Comment:
```suggestion
  color: ${({ primary }) => primary ? 'white' : 'palevioletred'};
```
이런 식으로 destructuring을 선행하면 코드를 줄일 수 있습니다.